openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -342,13 +343,16 @@ def test_patch_account_unknown_user(self, api_client, user):
         ("year_of_birth", 2009, "not_an_int", u"Enter a whole number."),
         ("name", "bob", "z" * 256, u"Ensure this value has at most 255 characters (it has 256)."),
         ("name", u"ȻħȺɍłɇs", "z   ", u"The name field must be at least 2 characters long."),
-        ("language", "Creole"),
         ("goals", "Smell the roses"),
         ("mailing_address", "Sesame Street"),
         ("bio", "Lacrosse-playing superhero"),
         ("bio", u"壓是進界推日不復女"),
         # Note that we store the raw data, so it is up to client to escape the HTML.
         ("bio", "<html>fancy text</html>"),
+        ("language_proficiencies", [{u"code": u"en"}, {u"code": u"fr"}], "not_a_list", {u'non_field_errors': [u'Expected a list of items.']}),
+        ("language_proficiencies", [{u"code": u"en"}, {u"code": u"fr"}], [{}], {'code': [u'This field is required.']}),
```

Comment:
Might as well add some diversity in the "success" cases. Single language, language that doesn't include "en" (since that is in the default profile), empty array, etc.